packages/next/src/build/swc/options.ts

```diff
@@ -82,6 +82,7 @@ function getBaseSWCOptions({
   bundleLayer?: WebpackLayerName
 }) {
   const isReactServerLayer = isWebpackServerOnlyLayer(bundleLayer)
+  const isAppRouterLayer = isWebpackBundledLayer(bundleLayer)
```

Comment:
maybe silly question but how does it being a webpack bundle layer determine that it's an app router layer?

Reply:
yea it used to called isWebpackAppLayer but renamed based on the behavior

Reply:
I created a new util `isWebpackAppPagesLayer` for now to make it sounds more matched with the usage